code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -237,8 +237,9 @@
 		/obj/item/device/multitool/hacktool,
 		/obj/item/reagent_containers/hypospray/combat,
 		/obj/item/stack/telecrystal,
-		/obj/item/device/radio
-		)
+		/obj/item/device/radio,
+		/obj/item/shield/riot/tact,
+		/obj/item/material/knife/tacknife)
```

Comment:
Put this bracket on the line below please.

Reply:
I don't understand what you mean. Do you mean the )? I don't see a bracket.

Reply:
Yes, the )

Reply:
Moved! :)